### PR TITLE
fix(NcCheckboxRadioSwitch): adjust to new design

### PR DESCRIPTION
**Pull request by marcoambrosini**
### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
Screenshot 2024-08-05 at 18 16 03 | Screenshot 2024-08-05 at 18 17 31

Fixes https://github.com/nextcloud-libraries/nextcloud-vue/issues/5875

### CODE REVIEW

**Comment by marcoambrosini** [comment]
@ reviewers the switch component is still 44px tall and I can't find out why.. .Do you have any ideas?

File: src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue [hunk 1]
```diff
@@ -682,13 +682,13 @@ export default {
...
 
-	$border-radius: calc(var(--default-clickable-area) / 2);
+	$border-radius: var(--border-radius-element);
 	// keep inner border width in mind
 	$border-radius-outer: calc($border-radius + 2px);
 
 	&--button-variant.checkbox-radio-switch {
 		background-color: var(--color-main-background);
-		border: 2px solid var(--color-border-maxcontrast);
+		border: 1px solid var(--color-border-maxcontrast);
```
Comment (susnux) [inline src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue:691]: Use the variable with fallback instead

File: src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue [hunk 2]
```diff
@@ -682,13 +682,13 @@ export default {
 		color: var(--color-primary-element-light);
 	}
 
-	$border-radius: calc(var(--default-clickable-area) / 2);
+	$border-radius: var(--border-radius-element);
```
Comment (susnux) [inline src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue:685]: Missing fallback

File: src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue [hunk 3]
```diff
@@ -198,7 +198,7 @@ export default {
 	gap: 4px;
 	user-select: none;
 	min-height: var(--default-clickable-area);
-	border-radius: var(--default-clickable-area);
+	border-radius: var(--border-radius-element);
```
Comment (susnux) [inline src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue:201]: Missing fallback

**Comment by ShGKme** [comment]
Sorry, I forgot about this PR when did https://github.com/nextcloud-libraries/nextcloud-vue/pull/5974

But border width change is still actual, I only fixed border-radius

**Pull request closed without merging**